pom.xml

```diff
@@ -158,6 +158,8 @@
         <hive.archive.download.skip>false</hive.archive.download.skip>
         <httpclient.version>4.5.14</httpclient.version>
         <httpcore.version>4.4.16</httpcore.version>
+        <hudi.version>0.14.0</hudi.version>
+        <hudi.spark.binary.version>3.4</hudi.spark.binary.version>
```

Comment:
let's update the comments for `<spark.version>`

Reply:
Done